_maps/RandomRuins/SpaceRuins/TheDerelict.dmm

```diff
@@ -5057,6 +5057,14 @@
 /obj/structure/cable,
 /turf/open/floor/plasteel/airless,
 /area/solar/derelict_aft)
+"oE" = (
+/obj/item/device/gps/flightdatarecorder{
+	gpstag = "Emergency Transponder - Kosmicheskaya Stantsiya 13"
```

Comment:
I was going to yell at you for not using actual cyrillic characters before realizing that BYOND sucks and you cant.